aanouncing-python-sdk-ga.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Announcing General Availability for Xata's Python SDK'
+description: 'We are excited to ship the first stable version of our Python SDK'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-go-logos.png
+  alt: Xata Python SDK
+author: Philip Krauss
+date: 08-28-2023
+published: true
+slug: announcing-the-python-sdk-ga
+---
+
```

Comment:
It would be nice to kick off this blog post with more details around why we're investing in a Python SDK generally. We could put together a narrative that is something like this:

- When we released Xata in a private beta our primary focus was on frontend users, investing in a really good TypeScript experience. But we also released a Python SDK that catered to a different audience
- Since that initial release, a few things have changed. The world has gone AI-crazy and Python has unofficially become the language of AI / ML. And, we released support for vector embeddings and our OpenAI integration for ChatGPT on your data
- With that perfect concoction, we've seen an uptick in our Python SDK adoption and lots of interest in a GA release

Something along those lines to help talk about the "why" around our investment in this SDK.

Reply:
Thank you for the inspiration, I knew a hook was missing, but didn't know how to start.